### PR TITLE
IPC-474 Registry value evaluation README note

### PR DESCRIPTION
**Pull request by Aerdayne**
See [this](https://github.com/nxt-insurance/application-service/pull/2512) for some context.
An ability to stub values resolved by registries with a clear API for unstubbing could help avoid cases like this in the future.
The API for stubbing a registry is the following:
```ruby
registry.enable_stubs! # This overrides the registry#resolve method
registry.stub(:a, :b, :c, 'value) # Stubs a value located at this path
registry.unstub # Unstubs all previously stubbed values
```

### CODE REVIEW

**Comment by getand** [comment]
Do we really need this? I mean the linked issue is actually due to other problems. And you can also simply use RSpec expect to receive and return if you want to stub this. Don't really see the point to introduce this tbh.

**Review by getand — COMMENTED** [review]
Don't really see how this will be useful very often as we did not really have the scenario for it yet.

**Comment by Aerdayne** [comment]
It's true that we can use RSpec's stubs for this. But in my opinion it's a bit cleaner if the library has an 'official' API for something.
If you're against this being merged, then I won't argue. It's truly not needed that much at this point. Perhaps in the future we might utilise this library for dependency injection, in which case we might not have access to the actual registry and this API could do all of the work under hood for us, but it's far fetched.
Since it's already been implemented in this PR and does not change any code I think there's no reason not to merge it.

**Comment by getand** [comment]
I just wanted to point this out. Don't think we will need it. And people would have to know this exists when they write a test. @lutfidemirci maybe you have an opinion. I would not merge it. But in the end your call

File: README.md
Comment (lutfidemirci) [inline README.md:287]: It is so great that you added this to README. Let's create a new PR for this if we decide to not merge this PR.

**Comment by lutfidemirci** [comment]
I am fan of not adding functionality that we do not need at the moment. This will increase maintenance cost without any immediate help.

File: README.md
Comment (Aerdayne) [reply README.md:271]: Purged this PR and left only this entry in the README.

**Review by lutfidemirci — APPROVED** [review]

**Pull request merged**
